### DISCUSSION TITLE
I am using deepexplainer for tabular data i dont know how much background data should be given!

### DISCUSSION THREAD

**Discussion by amos-coder** [shap/shap#3268] [Q&A]
I have a train set with 30 lakh samples,I want to get waterfall plot for my test set samples can someone suggest me how many background I should use?

I tried 1 lakh samples as my background data and got a waterfall plot but the problem is it's important features doesn't even matched 50% to the beesworm plot I got using 1 lakh data samples

**Comment by znacer** [comment]
Hi,
There is no clear rule that define what is best.  
The 2 common choice are to give the train dataset as a baseline or a whole part of a test set. In the last case, make sure the test set is not to biased.
